v2/devdoc/log_sink_console_requirements.md

```diff
@@ -0,0 +1,64 @@
+# `log_sink_console` requirements
+
+`log_sink_console` implements a log sink interface that logs the events to the console.
+
+## Exposed API
+
+```
+    extern const LOG_SINK_IF console_log_sink;
+```
+
+### log_sink_console.log_sink_log
+
+The signature of `log_sink_console.log_sink_log` is:
+
+```c
+typedef void (*LOG_SINK_LOG_FUNC)(LOG_LEVEL log_level, LOG_CONTEXT_HANDLE log_context, const char* file, const char* func, int line, const char* message_format, ...);
+```
+
+
+`log_sink_console.log_sink_log` implements logging to console.
+
+If `message_format` is `NULL`, `log_sink_console.log_sink_log` shall return.
+
+`log_sink_console.log_sink_log` shall obtain the time by calling `time`.
+
+`log_sink_console.log_sink_log` shall convert the time to string by calling `ctime`.
```

Comment:
How does the formatted time look like?

Reply:
ctime calls asctime and per the standard it does:

The asctime function converts the broken-down time in the structure pointed to by
timeptr into a string in the form
Sun Sep 16 01:03:52 1973\n\0